runtime/doc/eval.txt

```diff
@@ -3124,9 +3124,10 @@ text...
 				let lconst[0] = 2  " Error!
 				let lconst[1][0] = 'b'  " OK
 <							*E995*
-			|:const| does not allow to for changing a variable: >
+			It is an error to specify an existing variable with
+			:const.
 				:let x = 1
-				:const x = 2  " Error!
+				:const x = 1  " Error!
```

Comment:
Why `1` here? `2` was fine to show that const with the same name is not allowed.

Reply:
I changed it to show that the error has nothing to do with the value that was used.